lib/generators/komponent/install_generator.rb

```diff
@@ -53,6 +63,10 @@ def webpacker_default_structure
       def dependencies_not_met_error_message
         "Seems you don't have webpacker installed in your project. Please install webpacker, and follow instructions at https://github.com/rails/webpacker"
       end
+
+      def stimulus?
+        options[:stimulus]
```

Comment:
We should probably use the config here too, in case someone wants to define it in the config first so that he never has to pass the option.

Reply:
this is install generator, so its called on once on installation, so configuration not exist yet, so thats why I rely only on option

Reply:
The developer can add the configuration before running the install generator, can't he? Specifically to avoid passing the option.